mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -50,7 +50,7 @@ var generateCmd = &cobra.Command{
 	Use:   "generate",
 	Short: "Generate Models",
 	Long:  "Prerequisite: Excecute this command from the root of a meshery/meshery repo fork.\n\nGiven a Google Sheet with a list of model names and source locations, generate models and components any Registrant (e.g. GitHub, Artifact Hub) repositories.\n\nGenerated Model files are written to local filesystem under `/server/models/<model-name>`.",
-	Example: `
+	Example: `fu
```

Comment:
```suggestion
	Example: `
```

this seems like a typo?